### ISSUE TITLE
Is it possible for either it.backlink or it.filelink to open pdf file in Zotero pdf reader?

### ISSUE THREAD

**Issue by docsez** [PKM-er/obsidian-zotlit#230]
At the moment **it.backlink** opens the Zotero listing with the relevant entry in focus, and **it.filelink** opens the pdf in Preview rather than Zotero - so the highlights aren't easily visible.

Is there any way of adding a link to each annotation to take me to the relevant page of the pdf in Zotero where the annotation came from?

**Comment by aidenlx** [comment]
Yes, this is possible in v2. Each annotation now carries a link that opens directly to the matching page and highlight in Zotero's PDF reader, instead of just the general item listing or a separate PDF viewer. If it isn't already showing in your note template, you can add this link field to your annotation layout to display it.

v2 beta is available now: https://zotlit.aidenlx.site/docs/install-zotlit
Migration guide: https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
